README.adoc

```diff
@@ -57,19 +57,20 @@ server, then any logging software would do the trick.
 //include::{common-includes}/gitclone.adoc[]
 include::{common-includes}/gitclonedraft.adoc[]
 
-For the purposes of this guide, we are using Zipkin as our distributed tracing system. You can find the
-installation instructions for Zipkin https://zipkin.io/pages/quickstart.html[here]. You are not
-restricted to using Zipkin, but keep in mind that you may need additional instructions not listed here
-if you choose to use another tracing system.
+For this guide, use Zipkin as your distributed tracing system. You can find the installation instructions
+for Zipkin at the Zipkin https://zipkin.io/pages/quickstart.html[quickstart page]. You are not required
+to use Zipkin, but keep in mind that you may need additional instructions not listed here if you choose
+to use another tracing system.
 
-Before you proceed, make sure your Zipkin server is up and running. By default, Zipkin can be found at {zipkin-url}.
+Before you proceed, make sure your Zipkin server is up and running. By default, Zipkin can be found
+at the {zipkin-url} URL.
```

Comment:
I assume we're going to fill this in eventually?

Reply:
You mean {zipkin-url}? Its just a variable, it hold the url to zipkin and it will replaced automatically. This is so I dont accidentally make a mistake typing in the same url over and over.